packages/legend-graph/src/graphManager/protocol/pure/v1/transformation/pureProtocol/serializationHelpers/V1_DomainSerializationHelper.ts

```diff
@@ -66,11 +67,27 @@ export const V1_FUNCTION_ELEMENT_PROTOCOL_TYPE = 'function';
 export const V1_propertyPointerModelSchema = createModelSchema(
   V1_PropertyPointer,
   {
-    class: optional(primitive()),
+    owner: optional(primitive()),
     property: primitive(),
   },
 );
 
+/**
+ * @backwardCompatibility
+ */
+export const V1_deserializePropertyPointer = (
+  json: PlainObject<V1_PropertyPointer>,
+): V1_PropertyPointer => {
+  if (json.class) {
```

Comment:
I think to do this right, we should also check for the presence of `owner` field first else check for `.class` else throw `Can't deserialize property pointer`

Reply:
I think we don't want to throw an error here because `owner` field in `PropertyPointer` could be undefined so modified the code accordingly.

Reply:
I see, let's revert to what you have before then.

Reply:
done